File: DataStructure/Kotlin/src/Employee.kt
class Employee {
    public val personal    = Personal()
    public val salary      = Salary()

    class Personal {
        public var name : String = ""
        public var age : Int = 0
    }

    class Salary {
        public var raise : Boolean = false
        public var taxCode : TaxCode = TaxCode.LOWINCOME
    }
}

File: DataStructure/Python/employee.py
from taxcode import TaxCode

class Employee:
    def __init__(self):
        self.personal = self.Personal()
        self.salary = self.Salary()

    class Personal:
        def __init__(self):
            self.name = ""
            self.age = 0

    class Salary:
        def __init__(self):
            self.raise1 = False # raise is a reserved word
            self.taxCode = TaxCode.LOWINCOME